src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -393,6 +393,10 @@ def _delete_blob(blob_name):
     source_blobs = list(collect_blobs(client, source_container_name, pattern))
 
     if dryrun:
+        if if_modified_since:
+            logger.warning('--if-modified-since argument is ignored.')
+        if if_unmodified_since:
+            logger.warning('--if-unmodified-since argument is ignored.')
```

Comment:
I would be more specific here. `--foo argument is ignored when using --dry-run`.